lang/user/id.xml

```diff
@@ -562,24 +564,24 @@ Silakan download dan install firmware PS Vita terakhir.</no_modules>
 		<cpu>
 			<cpu_backend>Bagian Belakang CPU</cpu_backend>
 			<select_cpu_backend>Pilih backend CPU pilihan Kamu.</select_cpu_backend>
-			<cpu_opt>Aktifkan pengoptimalan</cpu_opt>
+			<cpu_opt>Aktifkan Pengoptimalan</cpu_opt>
 			<cpu_opt_description>Centang kotak untuk mengaktifkan optimasi CPU JIT tambahan.</cpu_opt_description>
 		</cpu>
 		<gpu>
-			<backend_renderer>Bagian Belakang Perender</backend_renderer>
+			<backend_renderer>Penyangga Grafis</backend_renderer>
 			<select_backend_renderer>Pilih perender backend pilihan Kamu.</select_backend_renderer>
 			<gpu>GPU (Harus ulang lagi aplikasinya)</gpu>
```

Comment:
<gpu>GPU dan kartu grafik (Harus ulang lagi aplikasinya)</gpu>
			<select_gpu>Pilih GPU dan kartu grafik Vita3K yang harus dijalankan.</select_gpu>

Reply:
keep the GPU, no need add it.

Reply:
ok but I think it's for a PC too, so I think it is just like that if there is a graphic card, but bro, think it's not good, I just come along, but I think it's like that

Reply:
okay, we need talk about in next pr, but no need add in this pr